backend/municipal-services/ingestion-service/app/utils/convertor.py

```
@@ -0,0 +1,224 @@
+    except Exception as e:
+        print(f"An unexpected error occurred: {e}")
+        return None
+
+
+def convert_json_to_boundary(json_str: str) -> List[Boundary]:
+    data = json.loads(json_str)
+    locations = [Boundary(**item) for item in data]
+    return locations
+
+
+def create_vendor_request(request_info:RequestInfo, vendor: Vendor):
```

Ruff (0.8.2)

Redefinition of unused `vendor` from line 7

(F811)